examples/book-and-change.py

```diff
@@ -2,12 +2,11 @@
 
 from duffel_api import Duffel
 
-
```

Comment:
My formatter did this.

Reply:
![](https://media.giphy.com/media/a2euXnuLIgVQA/giphy.gif)